pkg/cmd/root/root.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/coreeng/corectl/pkg/cmd/p2p"
 	"github.com/coreeng/corectl/pkg/cmd/template"
 	"github.com/coreeng/corectl/pkg/cmd/tenant"
-	"github.com/coreeng/corectl/pkg/cmd/update"
+	update "github.com/coreeng/corectl/pkg/cmd/update-cli"
```

Comment:
this confused me, I though this was the update command to update the config repositories.
It isn't, this is for updating the corectl binary